content/lessons/1_introduction/hello-world.rs

```diff
@@ -54,4 +54,40 @@ fn main() {
     }
 
     dbg!(x, y);
+
+    labels();
+
+    print!("{}", break_with_value() + 10);
+}
+
+fn labels() {
+    let mut count = 0;
+    'counting_up: loop {
+        println!("count = {}", count);
+        let mut remaining = 10;
+
+        loop {
+            println!("remaining = {}", remaining);
+            if remaining == 9 {
+                break;
+            }
+            if count == 2 {
+                break 'counting_up;
+            }
+            remaining -= 1;
+        }
+
+        count += 1;
+    }
+    println!("End count = {}", count);
+}
+
+fn break_with_value() -> i32 {
```

Comment:
I think it's better to show that the loop returns a value here through assigning that value to a variable instead of making it a return from a function